test/integration/orca_serve_offline_test.js

```diff
@@ -0,0 +1,84 @@
+const tap = require('tap')
+const Application = require('spectron').Application
+
+const { paths } = require('../common')
+
+const PORT = 9109 + 2
+
+const numberOfComponents = 6
+const axios = require('axios')
+const fs = require('fs')
+const pathToPlotlyJS = '/tmp/plotly-latest.min.js'
```

Comment:
Let's put that in `./build` aka [`paths.build`](https://github.com/plotly/orca/blob/2bee0c4333cf7327081f4852dde9c8b3c23e8e5c/test/common.js#L11) and use `path.join` so that your path is compatible on Windows:

```js
const = path.join(paths.build, 'plotly-latest.min.js')
```

Reply:
Brillant! Done in 3e107345b986a71601f8ff4cf845e7d9877463b1